src/components/Report.tsx

```diff
@@ -1,27 +1,79 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import DatePicker from "react-datepicker";
 import "react-datepicker/dist/react-datepicker.css";
 import { BsCalendarEvent } from "react-icons/bs";
 import "../styles/Report.css";
+import { DataStore } from "aws-amplify";
+import { Item } from "../models";
 
 export default function Report() {
   const [startDate, setStartDate] = useState<Date | null>(null);
   const [endDate, setEndDate] = useState<Date | null>(null);
+  const [totalSales, setTotalSales] = useState<number | null>(null);
+
+  useEffect(() => {
+    const calculateTotalValue = async () => {
+      let filteredItems = [];
+
+      if (startDate && endDate) {
+        filteredItems = await DataStore.query(Item, (c) =>
```

Comment:
I see you ended up using Datastore instead. In that case, can you delete all the new files that were created when you tried using graphql?

Reply:
I deleted the api.ts file and the graphql folder with the other 3.